_build/test/cover/aggregate/erl_cache_app.html

```diff
@@ -0,0 +1,199 @@
+<!doctype html>
```

Comment:
do not check in coverage. please )

Reply:
It's for convenience. There's no other easy way to share the report. I can remove the commit before merging, once approved.

Reply:
commit removed